server/api/order.spec.js

```diff
@@ -0,0 +1,73 @@
+/* global describe beforeEach it */
+
+//Needs to be finished!
+
+const {expect} = require('chai')
+const request = require('supertest')
+const db = require('../db')
+const app = require('../index')
+const Order = db.model('order')
+const Destination = db.model('destination')
+const Trip = db.model('trip')
+
+describe('Order routes', () => {
+  beforeEach(() => {
+    return db.sync({force: true})
+  })
+
+  describe('/api/order/:id', () => {
+    //make destinations
+    const bigBang = {
+      name: 'Big Bang',
+      imageUrl: null,
+      cost: 500,
+      timePeriod: 'Prehistoric',
+      decription: 'REALLY LOUD'
+    }
+
+    const frenchRev = {
+      name: 'French Revolution',
+      imageUrl: null,
+      cost: 100,
+      timePeriod: '19th Century',
+      decription: 'Really bloody... watch your head...'
```

Comment:
description needs an S! Sequelize will be angry. (The text below is supposed to let you just accept my suggestion, if I did it right???)

```suggestion
      description: 'REALLY LOUD'
```